index.html

```diff
@@ -0,0 +1,305 @@
+          <img src="./assets/icons/icn settings .icn-lg.svg" alt="briefcase icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">Digital marketing</h5>
+        </div>
+        <p class="feature-card__description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+      <div class="feature-card">
+        <div class="feature-card__head">
+          <img src="./assets/icons/ant-design_shop-twotone.svg" alt="shop icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">Digital marketing</h5>
+        </div>
+        <p class="description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+      <div class="feature-card">
+        <div class="feature-card__head">
+          <img src="./assets/icons/carbon_notebook.svg" alt="notebook icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">Business growing</h5>
+        </div>
+        <p class="description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+      <div class="feature-card">
+        <div class="feature-card__head">
+          <img src="./assets/icons/clarity_users-line.svg" alt="users icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">National top 50 firms</h5>
+        </div>
+        <p class="description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+      <div class="feature-card">
+        <div class="feature-card__head">
+          <img src="./assets/icons/bx_bx-stats.svg" alt="graph icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">Digital marketing</h5>
+        </div>
+        <p class="description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+      <div class="feature-card">
+        <div class="feature-card__head">
+          <img src="./assets/icons/fa-solid_award.svg" alt="medal icon" class="feature-card__icon">
+          <h5 class="headline headline--dark headline--bolder">Digital marketing</h5>
+        </div>
+        <p class="feature-card__description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="services">
+    <div class="section-text">
+      <h1 class="section-text__heading">We are providing best
+        business service.</h1>
+      <p class="description">We know how large objects will act, but things
+        on a
+        small scale just do not act that way.</p>
+    </div>
+    <div><img class="services__image" src="./assets/images/card-item.png" alt="video image"></div>
```

Comment:
Give a name to this class, all tags should have names

Reply:
Actually I don't even need the `div`
will remove it